swaglyrics/chrome.py

```diff
@@ -0,0 +1,14 @@
+# pyFile to keep Track of Global variables related to Chrome Extension.
+
+
+def initvariables():
+    global song, artist, isChrome, isTerminal
```

Comment:
not sure why isChrome is needed as you already have `args.chrome` with the same value and if the song and artist is returned through `spotify.py`, then you don't need `isTerminal` either

Reply:
True that. 

I treid it But it didnt seem to work.

 Beacuse inorder to get lyrics from browser, the server has to get started.

And once the server gets started, the code below doesnt get executed until the server is turned off

Reply:
Reason being, the `isTerminal` variable is intoduced. to handle the variables of same thread from` tab.py`